COLLABORATOR_GUIDE.md

```diff
@@ -471,9 +471,9 @@ $ git node land $PRID
 ```
 
 If it's the first time you ever use `node-core-utils`, you will be prompted
```

Comment:
I know it's not changed in this PR, but while you're here:

`you ever use` -> `you have used`